docs/tutorials/t2hahn_characterization.ipynb

```diff
@@ -238,8 +258,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Number of echoes\n",
-    "The user can provide the number of echoes that the circuit will perform. This will determine the amount of delay and echo gates. As the number of echoes increases, the total time of the circuit will grow. The echoes decrease the effects of $T_{1}$ noise and frequency inaccuracy estimation. Due to that, the Hahn Echo experiment improves our estimate for $T_{2}$. In the following code, we will compare results of the Hahn experiment with `0` echoes and `4` echoes. In addition, we will add frequency to the qubit and see how the result changes due to that (We can see Rabi Oscillations in the `0` echoes case).\n",
+    "### 3. Number of echoes\n",
+    "The user can provide the number of echoes that the circuit will perform. This will determine the amount of delay and echo gates. As the number of echoes increases, the total time of the circuit will grow. The echoes decrease the effects of $T_{1}$ noise and frequency inaccuracy estimation. Due to that, the Hahn Echo experiment improves our estimate for $T_{2}$. In the following code, we will compare results of the Hahn experiment with `0` echoes and `1` echoes. The analysis should fail for the circuit with `0` echoes. In order to see it, we will add frequency to the qubit and see how it affect the estimated $T_2$. <br>\n",
     "Note, that the provided delay time is the for each delay in the circuit and not the total time."
```

Comment:
Fix this line

Reply:
I don't know why this text is not changed. I already changed it when you last comment.

Reply:
Talking about the line starting with "Note", the sentence is corrupted

Reply:
changed to :
`Note, that the list of delays are the amount of time of each of the delay gates ($t_0$ in section 1).`